react-app/src/app/screens/Home/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import style from './styles.scss';
+
+class Home extends Component {
+  render() {
+    const renderLogged = logged => {
+      if (!logged) {
```

Comment:
I would leave this logic to the Route component. I wouldn't define a generic Home screen for each case, since they are different.